### ISSUE TITLE
Window is invisible/empty after update to Qt 6.7

### ISSUE THREAD

**Issue by nokia8801** [strawberrymusicplayer/strawberry#1421]
- [x] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [x] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [x] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
Application is running but the window is empty and nothing is visible.

**To Reproduce**
Upgrade to Qt 6.7 and launch the app.

**Expected behavior**
Window should be visible.

**System Information:**
 - Operating system: Arch Linux
 - Strawberry Version: strawberry 1.0.23.r112.g58944993
 - Qt 6.7.0 with qt6-wayland
 - GNOME 46.0 on Wayland

**Additional context**
Updated to Qt 6.7 on Arch Linux. Strawberry started crashing with segfaults. Compiled strawberry from the AUR again. Now it launches and there are no errors in GNOME Logs but the window is invisible and empty. Not even the header bar and menu is visible. Tried shortcuts thinking it may play a song but that didn't work either. Could very well be a Qt bug or Arch Linux bug but I don't know.

```
[nokia@linux strawberry-git]$ strawberry
06:56:25.562 INFO  main:182                         Strawberry 1.0.23-112-g58944993 Qt "6.7.0"
06:56:25.562 INFO  main:183                         "Arch Linux unknown - (linux 6.8.5-arch1-1) [x86_64]"
06:56:25.655 INFO  main:47                          TagReader worker connecting to "/tmp/strawberry_130699072"
06:56:25.896 WARN  GlobalShortcutsBackendGnome:92   Failed to grab media keys "org.freedesktop.DBus.Error.UnknownMethod" "Object does not exist at path “/org/gnome/SettingsDaemon/MediaKeys”"
06:56:26.123 INFO  DeviceManager:450                Device added: "/dev/sr0" QList("/dev/sr0")
```

**Comment by nokia8801** [comment]
Most likely a Qt or GNOME bug as all of my other Qt 6 apps are also invisible. It's visible when running with `QT_QPA_PLATFORM=xcb`.

**Comment by glaubersm** [comment]
Can confirm when running Strawberry on Gnome Wayland (other Qt 6 apps are affected too), cannot when running on KDE Plasma Wayland instead. I use Arch Linux.

**Comment by xuanruiqi** [comment]
Same - can only run under XWayland, not native wayland

**Comment by jonaski** [comment]
This seems like a general Qt/Wayland/Gnome specific issue, I suggest to report this bug to the Linux distribution.

**Comment by Pommede** [comment]
same issue, arch gnome wayland, but working with strawberry qt5...
its seems that we're alone ;-)

**Comment by nokia8801** [comment]
For anyone wondering, this issue is fixed by Mutter 46.1.
